.travis.yml

```diff
@@ -3,5 +3,7 @@ language: node_js
 node_js:
   - 4.0
   - 4
+  - 5.0
+  - 5
```

Comment:
Removed 5.0. I don't see why we shouldn't support 5 (and other non-lts releases). We didn't support 0.12 because it had a lot of issues. But it's not the case with 5 (hopefully).